mifosng-android/build.gradle

```diff
@@ -156,12 +156,12 @@ dependencies {
 
     //DBFlow dependencies
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    annotationProcessor "com.github.Raizlabs.DBFlow.dbflow:dbflow-processor:3.1.1"
-    annotationProcessor "com.github.Raizlabs.DBFlow:dbflow-processor:3.1.1"
+    annotationProcessor "com.github.raizlabs.dbflow.dbflow:dbflow-processor:3.1.1"
```

Comment:
what is the reason of renaming this?

Reply:
@danishjamal104 
[Link ](https://github.com/agrosner/DBFlow/issues/1321#issuecomment-307728901) for the reference

Reply:
cool